mandatory/3-function-output.js

```diff
@@ -1,17 +1,19 @@
-// Add comments to explain what this function does. You're meant to use Google!
-function getNumber() {
-  return Math.random() * 10;
+// Math.random() can be used to return random integers/numbers
+function getNumber(nim) {
+  return Math.random(nim) * 10;
 }
+console.log(getNumber(23))
 
-// Add comments to explain what this function does. You're meant to use Google!
+// .concat() is used to join two or more strings
 function s(w1, w2) {
-  return w1.concat(w2);
+  return w1.concat(w1, w2);
```

Comment:
I think if you use `w1.concat(w1, w2);` you will get a new string or array that will look like w1w1w2.
let's say you have var w1 = a, and var w2 = b, if you use the syntax like this you will get back as a return 'aab' instead of 'ab'.